app/assets/stylesheets/user-styles.css

```diff
@@ -28,10 +28,10 @@ img.right {
   margin: -8px; padding: 8px;
 }
 
-#sidebar .sidebar-title {
+#sidebar h3{
```

Comment:
this will apply to every h3 in the sidebar, are we concerned about there being other possible h3s?